index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Document</title>
+	<link rel="stylesheet" href="./style.css">
+</head>
+<body>
+	<div class="block1">
+		<div class="block2">
+			<div class="block3"> Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eveniet harum, distinctio illum delectus consequuntur ipsa cupiditate excepturi nemo esse ab? Itaque, voluptatum aliquid laudantium minus nam obcaecati unde autem impedit?</div>
+		</div>
+	</div>
+
+	<div class="block4">
```

Comment:
you don't need new classes for the next two blocks, it the same blocks like first and need to have the same classes
just create classes with the right styles for the first block, and then copy it 

remove all classes from `block4` to `block9`
you need to have only 3 classes, which will be reused